docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/configuring-low-resources.md

```diff
@@ -1,14 +1,13 @@
 ---
```

Comment:
Note to reviewers: this file has changed, so the doc URL will change too, but it's not a breaking change because it hasn't been released yet (the PR adding this file was just merged few hours ago).